code/modules/clothing/suits/storage.dm

```diff
@@ -62,3 +62,10 @@
 	pockets.max_w_class = ITEM_SIZE_SMALL
 	pockets.max_storage_space = 8
 
+/obj/item/clothing/suit/storage/vest/chestrig/New()
```

Comment:
``New()`` should always be put near declaration of actual object, most if not any procs should.

I don't know why it's like this for suit above. You might as well fix it.